en/config/safety.md

```diff
@@ -251,11 +251,10 @@ The parameters that control when the quad-chute will trigger are listed in the t
 
 Parameter | Description
 --- | ---
-<a id="COM_QC_ACT"></a>[COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) | Defines automatic action taken after a quad-chute. Can be set to: Warning only, Return, Land, Hold.
-<a id="VT_FW_QC_HMAX"></a>[VT_FW_QC_HMAX](../advanced_config/parameter_reference.md#VT_FW_QC_HMAX) |Maximum height above the ground (if available, otherwise above home if available, otherwise above the local origin) where triggering a quad-chute is possible. At high altitudes there is a big risk to deplete the battery and therefore crash if quad-chuting there.
-<a id="VT_QC_HR_ERROR_I"></a>[VT_QC_HR_ERROR_I](../advanced_config/parameter_reference.md#VT_QC_HR_ERROR_I) | Quad-chute uncommanded descent threshold. This is the threshold for integrated height rate error to trigger a uncommanded-descent quad-chute. Only checked in altitude-controlled fixed-wing flight. Additional conditions that have to be met for uncommanded descent detection are a positive (climbing) height rate setpoint and a negative (sinking) current height rate estimate.
-<a id="VT_QC_T_ALT_LOSS"></a>[VT_QC_T_ALT_LOSS](../advanced_config/parameter_reference.md#VT_QC_T_ALT_LOSS) | Altitude loss threshold for quad-chute triggering during VTOL transition to fixed-wing
- flight. Quad-chute triggers the current altitude is more than this value below the altitude at the beginning of the transition.
+<a id="COM_QC_ACT"></a>[COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) | Quad-chute action after switching to multicopter flight. Can be set to: [Warning](#action_warning), [Return](#action_return), [Land](#action_land), [Hold](#action_hold).
+<a id="VT_FW_QC_HMAX"></a>[VT_FW_QC_HMAX](../advanced_config/parameter_reference.md#VT_FW_QC_HMAX) | Maximum quad-chute height, below which the quad-chute failsafe cannot trigger. This prevents high altitude quad-chute descent, which can drain the battery (and itself cause a crash). The height is relative to ground, home, or the local origin (in preference order, depending on what is available).
```

Comment:
I modified this: VT_FC_QC_HMAX, please check.

I understand that this prevents quadchute triggering at height. My assumption is that this is a bit like an interlock - so if you have a quadchute fail at heigh altitude it gets ignored. When you get below this height if you are still in some other circumstance that causes a failsafe it will then trigger. However if you somehow recovered then it would not. Right? Do we need to capture this nuance?

Reply:
@hamishwillee If the vehicle goes below the threshold but the quadchute condition does not apply anymore, then it will not trigger anymore. So it's not like the FC remembers that it would have triggered and then triggers it once it's below the altitude.

Reply:
@hamishwillee 

> if you are still in some other circumstance that causes a failsafe it will then trigger.

Only if the circumstances are related to the conditions which trigger quadchute. Any other failsafe conditions like low battery has nothing to do with quadchute and thus will not trigger it.